saleor/checkout/__init__.py

```diff
@@ -6,15 +6,10 @@
 
 
 class AddressType:
-    BILLING = 'billing'
-    SHIPPING = 'shipping'
+    BILLING = "billing"
```

Comment:
It looks like black supports ignoring format strings as in https://github.com/ambv/black/issues/118#issuecomment-393298377. Since Saleor is using single quotes but black is using double qutoes, should we apply that?

Reply:
Had my doubts in the last project if I should use `--skip-string-normalization`. Wanted to have as 'standardized' config as possible to avoid discussions about what to choose.
Finally used default black setting (double quotes) and that turned out ok - I personally use single quotes but my editor change that without my attention. 
@maarcingebala what are your thoughts?

Reply:
I'd go with the default Black settings and use double quotes.